Samples/Sources/SampleDiningPhilosophers/Philosopher.swift

```diff
@@ -58,15 +60,19 @@ distributed actor Philosopher: CustomStringConvertible {
         }
 
         self.state = .thinking
-        self.timers.startSingle(key: .becomeHungry, delay: .seconds(1)) {
-            await self.attemptToTakeForks()
+        self.becomeHungryTimerTask = Task {
+            for await _ in AsyncTimerSequence(interval: .seconds(1), clock: ContinuousClock()) {
+                await self.attemptToTakeForks()
+                self.becomeHungryTimerTask?.cancel()
+                break
+            }
```

Comment:
Hmm okey those are just a "single" timers, so we can do these a bit simpler:

```
Task { 
  try await Task.sleep(until: .now + .seconds(1), clock: .continuous)
}
```

Reply:
Note that we also don't need the cancel then -- just run it once :)

Reply:
amended https://github.com/apple/swift-distributed-actors/pull/986/commits/892a857bf1854c548a2b07eb0e1134de7000ca44